source/server-discovery-and-monitoring/server-discovery-and-monitoring.rst

```diff
@@ -1264,10 +1264,9 @@ Logical Session Timeout
 
 Whenever a client updates the TopologyDescription from an ismaster response,
 it MUST set TopologyDescription.logicalSessionTimeoutMinutes to the smallest
-logicalSessionTimeoutMinutes value among all ServerDescriptions of
-known ServerType. If any ServerDescription of known ServerType has a null
-logicalSessionTimeoutMinutes, then
-TopologyDescription.logicalSessionTimeoutMinutes MUST be set to null.
+logicalSessionTimeoutMinutes value among ServerDescriptions of all ServerTypes
+except RSArbiter or RSGhost . If any have a null logicalSessionTimeoutMinutes,
```

Comment:
Extra space .

Reply:
Single-threaded SDAM includes the PossiblePrimary type, also shouldn't be trusted. And there's RSOther. How about we define "data node" to include mongos, primary, secondary, and standalone, and say we require a non-null session timeout from all data nodes?

Reply:
I agree we should only consider readable servers.

Reply:
Done